examples/08_tauri_paraview_https/config_app.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -x
+
+python -m PyInstaller \
+    --clean --noconfirm \
+    --distpath src-tauri \
+    --name server --hidden-import pkgutil \
+    --collect-data trame_vtk \
+    --collect-data trame_client \
+    --collect-data trame_vuetify \
```

Comment:
--collect-data on trame_* is not needed

Reply:
only if you don't use http for web delivery. Either way, you should not do http delivery.